<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>231</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>749</width>
    <height>231</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>智慧农业平台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>171</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100000</width>
      <height>100000</height>
     </size>
    </property>
    <widget class="QPushButton" name="ctl_auto_button">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>139</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>自动</string>
     </property>
    </widget>
    <widget class="QLabel" name="temp_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>39</y>
       <width>231</width>
       <height>42</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>环境温度：----</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>39</y>
       <width>223</width>
       <height>93</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="humi_label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>环境湿度：----</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="soil_label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>土壤湿度：----</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="fan_state">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>41</y>
       <width>113</width>
       <height>42</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>风扇：-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ctl_fan_button">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>48</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>开关</string>
     </property>
    </widget>
    <widget class="QLabel" name="pump_state">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>80</y>
       <width>113</width>
       <height>42</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>水泵：-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ctl_pump_button">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>87</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>开关</string>
     </property>
    </widget>
    <widget class="QPushButton" name="test_button">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>0</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>导出数据</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
